### PR TITLE
Start a separate RabbitMQ instance for each experiment's command queue

### PR DESCRIPTION
**Pull request by denkv**

### CODE REVIEW

**Comment by denkv** [comment]
@MichaelRoeder Is something wrong with Codacy?

File: platform-controller/src/main/java/org/hobbit/controller/PlatformController.java [hunk 1]
```diff
@@ -327,52 +302,24 @@ public void run() {
     }
 
     /**
-     * This method closes the command queue and its handling and reopens it on the
-     * RabbitMQ broker which will be used for experiments.
+     * This method sets the RabbitMQ connector for the command queue.
+     */
+    public void setExpRabbitMQConnector(RabbitMQConnector rabbitMQConnector) {
+        LOGGER.info("Setting experiment's RabbitMQ connector for the command queue: {}", rabbitMQConnector);
+        assert this.rabbitMQConnector == null : "RabbitMQ connector should be null";
```
Comment (MichaelRoeder) [inline platform-controller/src/main/java/org/hobbit/controller/PlatformController.java]: Using `assert` at runtime is not a good idea. Please throw an exception or try to handle the case in a different way.

File: platform-controller/src/main/java/org/hobbit/controller/PlatformController.java [hunk 2]
```diff
@@ -327,52 +302,24 @@ public void run() {
...
-                }
-            };
-            cmdChannel.basicConsume(queueName, true, consumer);
-        } finally {
-            // set the factory back
-            connectionFactory = tempFactory;
-        }
+    public void closeExpRabbitMQConnector() {
+        LOGGER.info("Closing experiment's RabbitMQ connector for the command queue: {}", rabbitMQConnector);
+        assert rabbitMQConnector != null : "RabbitMQ connector shouldn't be null";
```
Comment (MichaelRoeder) [inline platform-controller/src/main/java/org/hobbit/controller/PlatformController.java]: Again, an assert is not the best idea :wink:

**Comment by MichaelRoeder** [comment]
FYI: the codacy analysis for the develop branch was not activated. I fixed that.

**Pull request merged**